docs/en/advanced-concepts/chromedriver.md

```diff
@@ -0,0 +1,62 @@
+## Chromedriver
+
+Appium supports automating web pages (in Chrome and the built-in Browser) and
```

Comment:
Does it make sense to clarify that the "built-in Browser" is on Android? Or is that implied?